day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,21 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping
```

Comment:
@kevxo how would you update the data model in this method and the two below?

Reply:
I just updated everything through the Problem Fix commit and Error Commit